src/app/body/common-tools/suggestion-bucket/suggestion-bucket.component.html

```diff
@@ -22,9 +22,7 @@
                     {{userName}}
                 </span>
                 <br>
-                <span id="email">
-                    {{email}}
-                </span>
```

Comment:
Please don't remove emails from here rather receive an input from parent components if we want to show email or not

Reply:
Done !!